doc/man7/OSSL_PROVIDER-base.pod

```diff
@@ -57,25 +57,93 @@ currently permitted.
 
 The OpenSSL base provider supports these operations and algorithms:
 
+=head2 Random Number Generation
+
+The default provider all includes the "SEED-SRC" algorithm, which has the
+property "fips=yes", to allow it to be used together with the FIPS provider.
+
+=over 4
+
+=item SEED-SRC,  see L<EVP_RAND-SEED-SRC(7)>
+
+=back
+
 =head2 Asymmetric Key Encoder
 
-In addition to "provider=base", some of these encoders define the
-property "fips=yes", to allow them to be used together with the FIPS
-provider.
+The default provider also includes all of these encoding algorithms . Some of
+these have the property "fips=yes", to allow them to be used together with the
+FIPS provider.
 
 =over 4
 
-=item RSA, see L<OSSL_ENCODER-RSA(7)>
+=item RSA
+
+=item RSA-PSS
+
+=item DH
+
+=item DHX
 
-=item DH, see L<OSSL_ENCODER-DH(7)>
+=item DSA
 
-=item DSA, see L<OSSL_ENCODER-DSA(7)>
+=item EC
 
-=item EC, see L<OSSL_ENCODER-EC(7)>
+=item ED25519
+
+=item ED448
+
+=item X25519
+
+=item X448
+
+=item SM2
+
+=back
+
+=head2 Asymmetric Key Decoder
+
+The default provider also includes all of these decoding algorithms . Some of
+thesehave the property "fips=yes", to allow them to be used together with the
```

Comment:
Space betwix _these_ and _have_.

I'd avoid mentioning properties (here and above) and just say that some are configured to interact with the FIPS provider successfully or similar.